react/src/lib/components/DeckGLMap/layers/layersDefaultProps.ts

```diff
@@ -109,6 +109,7 @@ export const layersDefaultProps: Record<string, unknown> = {
         wellNameAtTop: false,
         wellNameSize: 14,
         wellNameColor: [0, 0, 0, 255],
+        trajectoryVisible: true,
```

Comment:
Do we need this if we can set the opacity of a well trajectory to 0?

Reply:
Currently as wellhead and well trajectory uses same color, in order to hide only the trajectory, we will need this. This setting is global, it applies to all the wells.

Reply:
What I mean is if we add `trajectoryColor` and `headColor`, then we can hide only the trajectories. For example 
```
trajectoryColor: [0, 0, 0, 0]
```